imagersite/imagersite/tests/test_home_page.py

```diff
@@ -0,0 +1,32 @@
+from django.test import TestCase
+from django.urls import reverse
+
+class HomePageTestCase(TestCase):
+
+    def setUp(self):
+        self.response = self.client.get('/')
+
+    def tearDown(self):
+        pass
+
+    def test_home_page_exists(self):
+        response = self.client.get(reverse('home'))
+        self.assertEqual(response.status_code, 200)
+
+    def test_for_registration_button(self):
+        """assert that the response contains a link to the registration page."""
+        reg_url = reverse('registration_register')
+        # expected = 'href = {}'.format(reg_url)
+        self.assertContains(self.response, reg_url, status_code=200)
+
```

Comment:
What about testing for the login button. What about testing for the random picture? What about testing differences between an authenticated user's view of the page and a non-logged-in user's view of the page?